src/app/system/account-number-preferences/edit-account-number-preference/edit-account-number-preference.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="accountNumberPreferenceForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Account Type</mat-label>
+            <input matInput formControlName="accountType" required>
+          </mat-form-field>
```

Comment:
Don't modify the control type. Keep select instead of input. You can disable it. Also, keep the error message.